batch_uploads_tarchive

```diff
@@ -1,4 +1,90 @@
 #!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+batch_uploads_tarchive - upload a batch of tarchives using script tarchiveLoader
+
+=head1 SYNOPSIS
+
+./batch_uploads_tarchive
```

Comment:
@cmadjar @nicolasbrossard 
shouldn't this have more details?:)

Reply:
The script actually does not take any arguments, switches or parameters...Once run, it takes its input from STDIN so you have to type the parameters *along the way*. The script supports -D and -v but if you look at the code, these switches are not used so I did not document them. So no I don't think this needs more details